src/fides/core/config/security_settings.py

```diff
@@ -30,6 +31,7 @@ class SecuritySettings(FidesSettings):
     """Configuration settings for Security variables."""
 
     root_user_scopes: Optional[List[str]] = SCOPE_REGISTRY
+    root_user_roles: Optional[List[str]] = [ADMIN]
```

Comment:
This is a new config value.  Thinking about it, there are a lot of places in the code where we assign the scope registry to a root user and now the admin role.  Instead, we might pull from this and the existing config value.

Reply:
Addressed in https://github.com/ethyca/fides/pull/2671/commits/ae8355b51fbcc48d014029ad80d8b40ae96e5fa6